src/styles/normalize.css

```diff
@@ -0,0 +1,17 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
you can't use Roboto if you don't import it - i don't see you did import it, so it does not apply

make sure to import the font